docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,51 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+### pmempool
+The pmempool is a management tool for Persistent Memory pool files created by PMDK libraries.
+DAOS uses the PMDK library to manage persistence inside ext4 files.
+[pmempool](https://pmem.io/pmdk/manpages/linux/v1.9/pmempool/pmempool-check.1/) can check consistency of a given pool file.
+It can be run with -r (repair) option which can fix some of the issues with pool file. DAOS will have more number of such pool file (vos-*), based
+on number of targets mention per daos engine. User may need to check each vos pool file for corruption on faulty pool.
+
+Below example, where system was not shutdown properly and that set the mode as dirty on the VOS pool file.
+#### pmempool check command ran on the system where the shutdown was not done cleanly
```

Comment:
Please shorten the title since it does not render well in material or on cell phone. Maybe something like "#### Unclean shutdown" and then provide a sentence describing the use case.

Please also add a blank line after each title, this is required by materials.